pkg/cli/cmd.go

```diff
@@ -95,6 +95,10 @@ func bindFlags(cmd *cobra.Command) {
 			val := viper.Get(f.Name)
 			cmd.Flags().Set(f.Name, fmt.Sprintf("%v", val))
 		}
+		// use CurrentProject as a backup for missing project flag
+		if !f.Changed && (f.Name == "project" || f.Name == "codeset-project") && viper.IsSet("CurrentProject") {
+			cmd.Flags().Set(f.Name, viper.GetString("CurrentProject"))
+		}
```

Comment:
The problem of doing it here is that I'll need to check which command is being executed, for example it's not possible to fill in "name" argument value, as it could be name of different artefact...